optuna/distributions.py

```diff
@@ -487,6 +517,11 @@ def check_distribution_compatibility(
 
     Returns:
         True denotes given distributions are compatible. Otherwise, they are not.
+
+    Raises:
+        ValueError:
+            If different distribution kind is set to ``dist_old`` and ``dist_new``,
+            or ``dist_old.choices`` doesn't match ``dist_new.choices``.
```

Comment:
Thank you for the quick review!

I have a question about that comment. 

If I add the sentence `for :class:`~optuna.distributions.CategoricalDistribution` `, I feel its meaning is to be a bit odd (might be from my lack of understanding regarding Optuna). Could you tell me your thought in detail?

Reply:
Sorry for the late reply.

I commented like this because the only `CategoricalDistribution` has the `choices` attribute. This error occurred only for `CategoricalDistribution`. Therefore, I think it's better to add "for :class:`~optuna.distributions.CategoricalDistribution`" here.

Reply:
Gotcha! I fixed. Thank you.